svgdigitizer/svgplot.py

```diff
@@ -1656,6 +1656,68 @@ def _min_real_root(cls, polynomial, tmin, tmax):
 
         return min(eligible_roots)
 
+    @property
+    def schema(self):
+        """A frictionless `Schema` object, including a `Fields` object
+        describing the voltage and current axis of the original plot
+        including original units.
```

Comment:
There's no voltage & current here.

Reply:
That's when you copy-paste without paying attention to the actual content... 🙈